package.json

```diff
@@ -8,6 +8,7 @@
         "build": "cross-env NODE_ENV=production webpack --config webpack.prod.js",
         "storybook": "cross-env NODE_ENV=development start-storybook -p 9001 -c .storybook --ci",
         "fakeapi": "json-server --watch fakeapi/db.json --routes fakeapi/routes.json --port 9002",
+        "dev": "npm-run-all --parallel start fakeapi",
```

Comment:
Let's some refactoring:
1. Rename `start` → `serve` and `start-local-api` → `serve-with-local-api`
2. Then rename `dev` → `start`
3. Fix npm-run-all command by point 1

So, at end we'll get short `yarn start` command for starting local developing

Reply:
yeah it's will be great to refactor like this.